experimental/uefi/channel/src/lib.rs

```diff
@@ -32,25 +33,76 @@ pub mod schema {
 }
 
 /// Length of the entire frame, including the header.
-pub type FrameLength = u32;
+type FrameLength = u32;
 const FRAME_LENGTH_SIZE: usize = 4;
 static_assertions::assert_eq_size!([u8; FRAME_LENGTH_SIZE], FrameLength);
 
-/// In request frames this field contains the id of the method_or_status that is invoked.
-/// In response frames this field contains the status code of the method_or_status invocation.
-pub type Method = u32;
+/// In request frames this field contains the id of the method that is invoked.
+/// In response frames this field contains the status code of the method invocation.
+type MethodOrStatus = u32;
 const METHOD_SIZE: usize = 4;
-static_assertions::assert_eq_size!([u8; METHOD_SIZE], Method);
+static_assertions::assert_eq_size!([u8; METHOD_SIZE], MethodOrStatus);
 
-pub const FRAME_HEADER_SIZE: usize = 8;
+// TODO(#2848): Use this space to send a stream identifier that identifies
+// frames as belonging to the same message. Currently the frames of every message
+// must be sent as an uninterrupted seqeuence. A stream identifier would allow us
+// to interleave messages.
+/// Reserved space to maintain 8 byte alignement.
+const SPACER_SIZE: usize = 4;
```

Comment:
Perhaps padding is a more common term than spacer for this sort of thing, but up to you.

Reply:
thx, updated